packages/devextreme/js/__internal/ui/chat/avatar.ts

```diff
@@ -82,10 +84,10 @@ class Avatar extends Widget<Properties> {
   }
 
   _updateAlt(): void {
-    const { name } = this.option();
+    const { alt, name } = this.option();
 
     // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-    this._$content?.attr('alt', name || messageLocalization.format('dxAvatar-defaultImageAlt'));
+    this._$content?.attr('alt', alt || name || messageLocalization.format('dxAvatar-defaultImageAlt'));
```

Comment:
May be it's better extract to separate variable to make code more readable? How do you think?

Reply:
[2675fac](https://github.com/DevExpress/DevExtreme/pull/28237/commits/2675facd58998531cb47d2ec39ebeffd7ba4a6c9)